app/models/player.rb

```diff
@@ -0,0 +1,84 @@
+class Player < ActiveRecord::Base
+  belongs_to :team
+
+  before_save :valid_outcomes_json?
+
+  validates :team, presence: true
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :batting_hand, presence: true
+  validates :stealing, presence: true
+  validates :running, presence: true
+  validates :vs_lhp, presence: true
+  validates :vs_rhp, presence: true
+  validates :avg, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :ab, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :doubles, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :triples, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :hr, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :rbi, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :bb, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :so, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :sb, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :cs, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :slg, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :obp, presence: true, numericality: { greater_than_or_equal_to: 0 }
+
+  enum batting_hand: {
```

Comment:
We need to figure out how to write unit tests for these. I'm okay punting on it for now but we should create a Github issue to remember to revisit it. Unfortunately the `define_enum_for` shoulda matcher seems to require the database to store values as integers, which is not what we want.